README.md

```diff
@@ -81,9 +81,14 @@ This is skipped if the requested file already has an extension.
 
 ##### index
 
-By default send supports "index.html" files, to disable this
-set `false` or to supply a new index pass a string or an array
-in preferred order.
+By default *send* supports "index.html" files. To disable this, set index option
+to `false`, or you can be able to supply a new index passing a string with the
+filename or an array in the preferred order.
+
+Alternatively, you can pass a function that will generate the index. The
```

Comment:
Rather than tacking this on as a separate paragraph, can you incorporate them all together? Otherwise, it sounds very hack in, but having a complete paragraph saying that the option only takes a boolean, string, or array, then another paragraph following to say "but wait! We forgot to add it takes a function too"